src/gregorio-utils.c

```diff
@@ -375,12 +375,20 @@ int main(int argc, char **argv)
     if (error_file_name) {
         gregorio_check_file_access(write, error_file_name, ERROR,
                 gregorio_exit(1));
-        error_file = freopen(error_file_name, "a", stderr);
+        /* First test if we can open error_file_name for writing, so
+           that if there is an error, we can still write the message
+           to stderr. */
+        error_file = fopen(error_file_name, "w");
```

Comment:
Probably need to normalize `error_file_name` before trying to open the error file.